### DISCUSSION TITLE
Which is higher? mmocr1.0.0rc4 or mmocr1.0.0?

### DISCUSSION THREAD

**Discussion by zhongqiu1245** [open-mmlab/mmocr#1916] [General]
Hello dear authors!
Thank you for your amazing job! 
When I want to deploy mmocr in mmdeploy, mmdeploy requires mmocr >=mmocr1.0.0rc4. So which is higher? mmocr1.0.0rc4 or mmocr1.0.0 in Tag? 
Thank you

**Comment by gaotongxiao** [comment]
1.0.0 is the latest one.

**Comment by zhongqiu1245** [comment]
thx!
